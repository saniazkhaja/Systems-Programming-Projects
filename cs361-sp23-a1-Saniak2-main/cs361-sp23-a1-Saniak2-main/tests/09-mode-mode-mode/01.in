054aba0a9477ef6a
