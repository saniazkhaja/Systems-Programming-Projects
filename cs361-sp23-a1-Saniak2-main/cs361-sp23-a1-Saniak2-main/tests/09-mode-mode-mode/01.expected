validation function failed
235275606
